tools/corda-runtime-gradle-plugin/src/main/kotlin/net/corda/gradle/plugin/cordapp/CordappTasksImpl.kt

```
@@ -27,9 +41,11 @@ class CordappTasksImpl(var pc: ProjectContext) {
     fun createGroupPolicy() {
         val groupPolicyFile = File(pc.groupPolicyFilePath)
         val networkConfigFile = File("${pc.project.rootDir}${pc.networkConfig.configFilePath}")
+        if (pc.networkConfig.mgmNodeIsPresentInNetworkDefinition) {
```

(not a blocker, but nice to have. your call to address in this PR or not) It's a bit confusing that this task just silently does nothing in this case. It would be more obvious to only assume it is being depended on when static network is required, and skipped for dynamic network.

Also, as it doesn't make sense to run it for the dynamic network scenario, then an error/warning message to user would be nice.